src/messages/MMgrReport.h

```diff
@@ -106,6 +108,8 @@ class MMgrReport : public MessageInstance<MMgrReport> {
   // encode map<string,map<int32_t,string>> of current config
   bufferlist config_bl;
 
+  OSDPerfMetricReport perf_report;
```

Comment:
Nit: ```osd_perf_metric_report``` perhaps to avoid ambiguity.

Reply:
Fixed in: https://github.com/ceph/ceph/pull/24215/commits/3a5f9989c9b6402152b475ba295d4b88c2470906